distribution/kernel/carbon-home/repository/resources/conf/templates/repository/conf/registry.xml.j2

```diff
@@ -27,9 +27,13 @@
     <enableCache>true</enableCache>
     <registryRoot>/</registryRoot>
 
-    <dbConfig name="wso2registry">
+    {% if registry.local_carbon_db_enabled is sameas true %}
```

Comment:
how about database.wso2_carbon_db.enabled instead? such that it will be consistent with the already defined db configs.

Reply:
+1